src/encoded/types/antibody_lot.py

```diff
@@ -158,151 +162,166 @@ def lot_reviews(characterizations, targets, request):
         target_organisms[organism] = target['@id']
     target_organisms['all'] = tmp
```

Comment:
It would be a little more succinct to do:
`target_organisms = { 'all' : [] } `
and then append to that list in the targets loop